### ISSUE TITLE
There's a space between the type of an array and the square brackets: 'int[]' becomes 'int []'

### ISSUE THREAD

**Issue by reijerh** [fsprojects/fantomas#2192]
Issue created from [fantomas-online](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKAbRAXMAzWsYAFAEYCGATmgJYB2WA2gLoCUYAvADoWe0QS4y1dAHdqWABZhOkfLBmQANOCgBnbFjoBzVajAMVEUP36QAJFgCeAB2RpIdLEsOmzANzLoArnf0uTAAzKJiaQAJK0SPQAytQAXsjBIRAALP5gTC4AvklG6eZWtnoO9M7J5h7evgbJEACMuSGQALJkAB4AMnSIHYi0WpJltXUATAHpmSY5LsZNFja%2BkH3wAPK4XbSI0VaYQ6Hunj56NbUjjaEAopFrG4npEJDouJAT2Y2z%2B4WLJAToe66VI5oE7JADM51MEXUFCwADE6J4AHKIETobr-ExYCg%2BV5Td75eZFew-WB-CEVQ7Ve5gFIQh7RRAAW2ocHQsFoAEEsFdVut0XS8J51Lj%2BNMTB9XF9iiSyQTAVTamAAKwCyDRaxkaCIABCiHwFEQAAVKGRGdhEBQMfwsTjkpNRfjyoTvr8rRSqsdqQA2VXqzU6vWwA0dWAiC3QMjqCJuWARzTsq0QG2IEUQMX8CUPZ3S13kg4e4HUgDsvo1Wt1%2BsQAFVrLYKBGo7QY3HqAmBYJ0MK7W8ZgSpcTc3LKZ7FQAOUv%2BitBxAAYXQkdUM-Zqht0CwQcTgs7Ke7eN7Tv7UBlbvzQL8ioAnBPy4GDc0mSQLZuO12Qva0465ofIMe8-KR8MQTUmqZYBpWS5srQz5Cjub49uKfYLDmpInv%2BhaKnUDTAX6WocrgWAWkujKMmQm7JqmYDpnkB5IQOKF-sO6HDGc2GgVOBoMsyrJttSL6wSY76UZ%2Bny0Ueg5Omh57DOCrH%2BnhBEUJxLKkjxirkbuDr7l%2Bok-uJcySSCIR1LSwERFEWArLQAAqFCWAA6uIEjQduFFUWAmZQNmdGyhJjFSckdQqrJuEULAXiRAAIogaLMgpZHYvx-CCW5HkFDpjioX5hkmHUPrAa0bRhLgVkSH0FzbtEEhBlgDnwIMAopOMGkflpIlElAGUMQW-lGSW%2BXtBEuDUG0Kx1mQ64UBcbTWAaqiqK2UECkqTVwXuCE0e1JROF1Z7ZfwdTjv1bQAEqIHAFDwLV9XUo1rnCZK6WlDtCrDFeR2ned8CIl4jKPhQaxhARjK6AKdR3a1D2bYyXjoJoaKbLCQYkVgClWUhz0AckYyqh9QbwM0MNw90iMUMjcWqtAEgmmuFoAPpiHVTng%2Bt2mbZ1Q7dXtEAjFhiotO0HIUBQZCWCsFBdCuV1OQ1K0CfBGaIWzT0c7t1IjCxfMFYLwui%2BL1Art9v0WgDQMg9SYPNUJENZt%2B0Ow8IxNI%2BNqPoyrL1YzJfPayLYsS1gBP2-DiAk2TT4U1Tws0xQ9PUIzLyWyliuLOzvmc2rJma%2B0ABqlLanQ8DaFLm6jrLSXy9RrPJ8rqeq4qIxBZnbSwuFa4LXnkSF7H12KrdCf3Tbj3bW7mMhCMeWNxFsBYAA4tgU0zYgc0LUXS2lxAyX95536-sPTFY31muEw7mzamy0AANbapH5-YKolnRKas6kj9i28TBzMKxtLr0bvPUmCMh0%2BbIlRN0XUWAwx9DRrYKKQ1aDiAWhySI94jYUDNoqPiH8K5tW-j5fSWU1ZvT5gAaUQIgawRUSp9AiA-M0txnKvjlmtT%2BldigpzwWnRUoIgKN3KsyVQEgV43TXhkcu7kk7IVwfsAy1JQS81qGqbQmBBZaB%2Bn0GqiASDNAQHcdB78%2B7Wy3jpHeNd3YhFBBreRHI0RaFoM3Wgrd2SxBseNLwBorKwDMmo8aC16GJXXqI1KXkxI-xMSPEwoJPaWNhhaWg3i3A9HZFoZBf0YF0HgcuXxmCxFfwkZlDhtRQQZ3kQHTQtwOimhIPAMgc5YDzX6MAoOmT9Es2wbkjGe8zEN3kRFfWZASCYF4frCQ0RLD0HaE01amkWmQxwXk2uBTx7yJIWQzx9AIhXzIPY6Wb8XLNOYa07yczTHhIPvI0%2Bmzz63FUILMKkRkQrkQPjI%2BQd56zXmhkgU6lJktWmQPTaxj2HzLBIAs5lB2KIB6aoaAFBqDMliQReAVY4HsiitAecwt4yv10bs75VtfmGP%2BXpKR%2BDOGEPkcdMgzjQrnwmYwqZ%2ByZltN-lzGk3D5HbBhWuLRSBaVlymCoJgSQHCqFhPNPQGCshAA)

#### Code

```fsharp
let foo (bar: int[]) =
    failwith "foo"
```
            

#### Result

```fsharp
let foo (bar: int []) = failwith "foo"

```
            
#### Problem description

The square brackets of array types are separated from the type by a space. I've never seen this style in .NET land, the style guide says:
> For F# arrays, use the syntactic name int[] or int array rather than array<int>.

#### Extra information

- [ ] The formatted result breaks by code.
- [ ] The formatted result gives compiler warnings.
- [ ] I or my company would be willing to help fix this.

#### Options

Fantomas master branch at 2022-04-04T14:08:13Z - e784cd68954e7c6fac13ed723d108920d04c1ecc

Default Fantomas configuration

<sub>Did you know that you can ignore files when formatting from fantomas-tool or the FAKE targets by using a [.fantomasignore file](https://github.com/fsprojects/fantomas/blob/master/docs/Documentation.md#ignore-files-fantomasignore)?</sub>

**Comment by nojaf** [comment]
Hello, thank you for bringing this up.
It appears we are not following the style guide here.
Are you interested in addressing this?
A PR can be submitted against the [get-back](https://github.com/fsprojects/fantomas/tree/get-back) branch.

**Comment by nojaf** [comment]
Great, a good starting point would be our [Contribution Guidelines](https://github.com/fsprojects/fantomas/blob/master/CONTRIBUTING.md).

**Comment by dsyme** [comment]
Looks like this can be closed?  Was addressed in #2192? :)

**Comment by nojaf** [comment]
Yes, this wasn't automatically closed because that PR wasn't targeting the master branch.
